### ISSUE TITLE
Sliders for Integration, Gain, Scan Avg

### ISSUE THREAD

**Issue by samiebee43** [WasatchPhotonics/EnlightenMobile#16]
<img src="https://github.com/WasatchPhotonics/EnlightenMobile/assets/124081765/98d6cee3-a085-4b87-8a98-02fb924747f0" width=40%>
<img src="https://github.com/WasatchPhotonics/EnlightenMobile/assets/124081765/69e3a28e-7dc8-4b2b-9901-53dcda84091a" width=40%>

* The layout is already cramped without the keyboard, and the keyboard is difficult to dismiss.
* [Lengthy article about sliders](https://www.smashingmagazine.com/2017/07/designing-perfect-slider/#enhancing-a-slider-with-precise-inline-editing)
* TLDR exact inputs for INT, GAIN, AVG could be placed in the "Device" tab or similar if we need them.

**Comment by samiebee43** [comment]
Sliders have been added.
Need to connect them to labels and spec.

**Comment by samiebee43** [comment]
Sliders are connected to labels and spec. Default values are misbehaving, and slider xaml parameters [are fairly broken](https://github.com/xamarin/Xamarin.Forms/issues/1943#issuecomment-431508922).

I can work around this with the following change.

App starts on [ Device ] tab and requires user to Connect before going to the [ Scope ] tab. Upon connection I will programmatically set the sliders to match on-device settings -- (instead of using load-time defaults).

**Comment by samiebee43** [comment]
This is already an isse #24
